app/src/main/res/layout/fragment_cryptocurrencies_available.xml

```diff
@@ -0,0 +1,124 @@
+        android:text="@string/markets"
+        android:textColor="@color/text_light"
+        android:textSize="@dimen/text_size_header"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <CheckBox
+        android:id="@+id/market_mxn_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_small"
+        android:text="@string/market_mxn"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/markets_txt"
+        style="@style/style_check_box" />
+
+    <CheckBox
+        android:id="@+id/market_bitcoin_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/market_bitcoin"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/market_mxn_checkbox"
+        style="@style/style_check_box"/>
+
+    <CheckBox
+        android:id="@+id/market_usd_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/market_usd"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/market_bitcoin_checkbox"
+        style="@style/style_check_box" />
+
+    <CheckBox
+        android:id="@+id/market_dai_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/market_dai"
+        app:layout_constraintStart_toStartOf="@+id/guideline2"
+        app:layout_constraintTop_toTopOf="@+id/market_mxn_checkbox"
+        style="@style/style_check_box" />
+
+    <CheckBox
+        android:id="@+id/market_arg_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/market_arg"
+        app:layout_constraintStart_toStartOf="@+id/guideline2"
+        app:layout_constraintTop_toBottomOf="@+id/market_dai_checkbox"
+        style="@style/style_check_box" />
+
+    <CheckBox
+        android:id="@+id/market_brl_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/market_brl"
+        app:layout_constraintStart_toStartOf="@+id/guideline2"
+        app:layout_constraintTop_toBottomOf="@+id/market_arg_checkbox"
+        style="@style/style_check_box" />
+
+    <TextView
+        android:id="@+id/cryptos_txt"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_small"
+        android:layout_marginTop="@dimen/margin_huge"
+        android:fontFamily="@font/montserrat_bold"
+        android:text="@string/cryptocurrencies"
+        android:textColor="@color/text_light"
+        android:textSize="18sp"
```

Comment:
Preferably, all these textSizes and dimensions should be referenced from the `dimens.xml` file. That way you can reuse them wherever needed.